src/components/header/index.njk

```diff
@@ -32,7 +32,7 @@ The [ghost button](/components/button/#header-style) in the header moves to the
 
 The external header with language switch and save button, should be used across public facing services.
```

Comment:
Move this above the example. We usually explain the variant before showing the example